AricHasting/senior-software
# Chat

# PR for Card \#https://trello.com/c/wHXClrNT/11-server-client-connection-between-user-agent


[Card #XX](https://trello.com/c/wHXClrNT/11-server-client-connection-between-user-agent)

## Goal/Problem
Not everything was modular.

## Solution
Everything is a method and ready to be used by a GUI

## Verification Steps
Call startserver(HOST, PORT), then call connect(HOST, PORT) for two different clients. Use send(msg), has_message(), and receive(msg) to verify the chatroom works

## Notes
